hangmanlunch.rb

```diff
@@ -0,0 +1,208 @@
+require "colorize"
+
+class Hangman
+
+  def initialize
+    @answer_array = populate_answer
+    @guess_array = populate_guess_array
+    @paper = []
+    @top_row =      "|    _________"
+    @second_row =   "|     |/      "
+    @middle_row =   "|     |       "
+    @base_row =     "| ____|___ word: #{@guess_array.join(" ")}"
+    @misses = 0
+    @incorrect_array =[]
+
+    #remove this line to actually play-- leaving it for testing purposes
+    puts "Answer: " + @answer_array.join
+
+    start
+  end
+
+  # create paper
+  def draw_paper
+    @paper.each do |row|
+      puts row
+    end
+  end
+
+  def start
+    puts "\nLet's play Hangman!!\n"
+
+    #this kicks off the game loop in "play" method below
+    @now_playing = true
+
+    # creates empty scaffold (in the paper array)
+    @paper.push(@top_row, @second_row, @base_row)
```

Comment:
As far as I can tell, you're not using the `@top_row` or `@second_row` instance variables anywhere else. If you ever wanted to refactor, you could likely pull this line and the loop below into a `build_paper` method and not use instance variables.

In general, if you only use it once, default to using local variable; you can always instance-fy it later.